sensor/kubernetes/listener/resources/secrets.go

```diff
@@ -350,6 +379,23 @@ func (s *secretDispatcher) processDockerConfigEvent(secret, oldSecret *v1.Secret
 	return events
 }
 
+// skipIntegrationCreate returns true if image integrations should NOT be created from this secret.
+func skipIntegrationCreate(secret *v1.Secret) bool {
+	if features.SourcedAutogeneratedIntegrations.Enabled() {
+		// Create all integration if sourced autogen enabled.
+		return false
+	}
+
+	if env.AutogenerateGlobalPullSecRegistries.BooleanSetting() &&
+		secret.GetNamespace() == "openshift-image-registry" && secret.GetName() == "installation-pull-secrets" {
+		// This secret is a copy of the OCP global pull secret, managed by the cluster-image-registry-operator,
+		// skip to avoid unecessary dupes.
```

Comment:
is there a link to documentation or something that explains this is a copy?

Reply:
I'm not aware of any docs, saw it when testing, and then later found it in code: https://github.com/openshift/cluster-image-registry-operator/blob/2e5523193fdfd78976ea1507b5e3f470ed12e6b6/pkg/resource/pullsecret.go#L51-L53

Reply:
I'd say we should use the constants they defined https://github.com/openshift/cluster-image-registry-operator/blob/5ff519f0ef47a74e7bba80a5d76e216c60cb26df/pkg/defaults/defaults.go#L16 but that means pulling in another dependency. Probably not worth it...

Would you mind linking this file (here's the latest release branch - https://github.com/openshift/cluster-image-registry-operator/blob/release-4.20/pkg/resource/pullsecret.go) here?